.github/workflows/deploy-ec2.yml

```
@@ -0,0 +1,97 @@
+name: Deploy to EC2
+
+on:
```

YAMLlint (1.38.0)

[warning] truthy value should be one of [false, true]

(truthy)